recipe/meta.yaml

```diff
@@ -442,7 +445,7 @@ outputs:
       skip: True  # [osx]
       detect_binary_files_with_prefix: false
       run_exports:
-        - {{ pin_subpackage('impi_rt') }}
+        - {{ pin_subpackage('impi_rt', min_pin="x.x.x", max_pin="x.x") }}
```

Comment:
Isn't this too strict? Perhaps min_pin=x.x and max_pin=x would be enough?

Reply:
from intel packages perspective the expectation that there is no mix of different devel and _rt packages - they supposes to be same version and get deployed together. 

However real compatibility is broader.  What are negative implications of too strict deps here? 
Sorry for naive question

Reply:
Oh, sorry, I got lost in context.  
Now that I realize this is impi-devel pinning impi_rt, I have to ask: 
Should it be using `pin_subpackage('impi_rt', exact=True)` instead?
If it does not make sense, then please just ignore my comments/questions and move forward.

Reply:
I think ether would work. Unless there are some additional considerations  that i do not understand =)

Reply:
Or at least maybe `pin_subpackage('impi_rt', min_pin="x.x.x", max_pin="x.x.x")`, with min_pin equal to max_pin.

Relavant docs [here](https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html#referencing-subpackages).

Reply:
exact pinning goes down to the build string which seems too strict here. 

the negative side effects of too tight pinnings are basically that it makes environments harder to solve and more fragile. 

my suggestion is `pin_subpackage('impi_rt', min_pin="x.x.x", max_pin="x.x.x")` or equivalently `impi_rt =={{ version }}`